### PR TITLE
Added readme information under /docs about building and prerequisites

### PR DESCRIPTION
**Pull request by ginabeena**
Just the beginning :) the first PR.

### CODE REVIEW

**Comment by ginabeena** [comment]
@esfoobar-usds @TomNUSDS what do you think about me adding some `.vscode/tasks.json` so that we can get a project view and IDE build actions. Not sure what y'all use for an IDE.

**Review by TomNUSDS — APPROVED** [review]
lgtm

File: docs/README.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
+# Installation Prerequisites
+
+1. [Install the latest docker](https://docs.docker.com/get-docker/) and you may want to install [docker desktop - latest](https://docs.docker.com/desktop/)
+2. [Install python 3.9.5 (latest) and](https://www.python.org/downloads/) [configure your environment](https://docs.python.org/3/tutorial/venv.html) to get data-platform source environment ready.
+3. Install Visual Studio Code (python packages & typescript) <TODO: add .vscode>
+4. [Installing nvm](https://github.com/nvm-sh/nvm#installing-and-updating) and then do:
```
Comment (NatHillardUSDS) [inline docs/README.md:6]: Other steps have `Install` as opposed to `Installing`

File: docs/README.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
+# Installation Prerequisites
+
+1. [Install the latest docker](https://docs.docker.com/get-docker/) and you may want to install [docker desktop - latest](https://docs.docker.com/desktop/)
+2. [Install python 3.9.5 (latest) and](https://www.python.org/downloads/) [configure your environment](https://docs.python.org/3/tutorial/venv.html) to get data-platform source environment ready.
```
Comment (NatHillardUSDS) [inline docs/README.md:4]: This overlaps a bit with the in-progress python instructions in [this](https://github.com/usds/justice40-tool/pull/122)  PR -- we should reconcile at some point

**Review by NatHillardUSDS — COMMENTED** [review]
A few small comments otherwise good; I think the build failure was temporary and should go away on rerun

File: docs/README.md (hunk 2 above)
Comment (ginabeena) [reply docs/README.md:4]: Yeah, do I need to change the version of python? I think for right now let's just get this merged and we can update later

File: docs/README.md (hunk 1 above)
Comment (ginabeena) [reply docs/README.md:6]: yeah i'll fix when i update the python code

File: docs/README.md (hunk 2 above)
Comment (NatHillardUSDS) [reply docs/README.md:4]: yeah makes sense to merge now then adjust as the other evolves

**Review by NatHillardUSDS — APPROVED** [review]

**Pull request merged**